packages/client/hooks/useGroupMatrix.ts

```diff
@@ -19,9 +19,10 @@ const useGroupMatrix = (
     const width = el?.clientWidth
     if (!width) return null
     let colCount = 1
+    const currentMaxColumns = Math.min(resultsGroups.length, MAX_SPOTLIGHT_COLUMNS)
```

Comment:
Fixes https://github.com/ParabolInc/parabol/issues/5625. If there's only enough reflections for 1 or 2 columns, we don't want to return 3 columns